template/nodejs/config.json

```diff
@@ -0,0 +1,3 @@
+{
+  "token": ""
+}
```

Comment:
@AwesomeStickz Consider using .env instead of config.json?

Reply:
Uses .env now